packages/preset-create-react-app/index.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const { logger } = require('@storybook/node-logger');
+const mergePlugins = require('./helpers/mergePlugins');
+const reactScriptsPath = require('./helpers/reactScriptsPath');
+const processCraConfig = require('./helpers/processCraConfig');
+const checkPresets = require('./helpers/checkPresets');
+
+const SCRIPTS_PACKAGE_OPTION = 'scriptsPackageName';
+
+checkPresets();
+
+const resolveLoader = {
+  modules: ['node_modules', path.join(reactScriptsPath, 'node_modules')],
+};
+
+// Ensure that loaders are resolved from react-scripts.
+const managerWebpack = (webpackConfig = {}) => ({
+  ...webpackConfig,
+  resolveLoader,
```

Comment:
why is this needed for the manager?!

🤔 🤔 🤔 🤔 🤔 🤔 🤔 

This is my only real concern from this PR

Reply:
Good question. This is the same solution we use now in `@storybook/react` and it allows us to use the `babel-loader` that `react-scripts` ships with - so we avoid conflicts with that version.